website/docs/guides/eks-getting-started.html.md

```diff
@@ -0,0 +1,594 @@
+sed -i s,MASTER_ENDPOINT,${aws_eks_cluster.demo.endpoint},g /var/lib/kubelet/kubeconfig
+sed -i s,CLUSTER_NAME,${var.cluster-name},g /var/lib/kubelet/kubeconfig
+sed -i s,MASTER_ENDPOINT,${aws_eks_cluster.demo.endpoint},g /etc/systemd/system/kubelet.service
+sed -i s,MASTER_ENDPOINT,${aws_eks_cluster.demo.endpoint},g /etc/systemd/system/kube-proxy.service
+sed -i s,INTERNAL_IP,$INTERNAL_IP,g /etc/systemd/system/kubelet.service
+sed -i s,INTERNAL_IP,$INTERNAL_IP,g /etc/systemd/system/kube-proxy.service
+sed -i s,CERTIFICATE_AUTHORITY_FILE,$CA_CERTIFICATE_FILE_PATH,g /var/lib/kubelet/kubeconfig
+sed -i s,CLIENT_CA_FILE,$CA_CERTIFICATE_FILE_PATH,g  /etc/systemd/system/kubelet.service
+systemctl daemon-reload
+systemctl restart kubelet kube-proxy
+USERDATA
+}
+
+resource "aws_launch_configuration" "demo" {
+  associate_public_ip_address = true
+  iam_instance_profile        = "${aws_iam_instance_profile.demo-node.name}"
+  image_id                    = "${data.aws_ami.eks-worker.id}"
+  instance_type               = "m4.large"
+  name_prefix                 = "terraform-eks-demo"
+  security_groups             = ["${aws_security_group.demo-node.id}"]
+  user_data_base64            = "${base64encode(local.demo-node-userdata)}"
+
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+```
+
+Finally, we create an AutoScaling Group that actually launches EC2 instances
+based on the AutoScaling Launch Configuration.
+
+~> **NOTE:** The usage of the specific `kubernetes.io/cluster/*` resource tag below is required for EKS and Kubernetes to discover and manage compute resources.
+
+```hcl
+resource "aws_autoscaling_group" "demo" {
+  desired_capacity     = 2
+  launch_configuration = "${aws_launch_configuration.demo.id}"
+  max_size             = 2
+  min_size             = 1
+  name                 = "terraform-eks-demo"
+  vpc_zone_identifier  = ["${aws_subnet.demo.*.id}"]
+
+  tag {
+    key                 = "Name"
+    value               = "terraform-eks-demo"
+    propagate_at_launch = true
+  }
+
+  tag {
+    key                 = "kubernetes.io/cluster/${var.cluster-name}"
+    value               = "owned"
+    propagate_at_launch = true
+  }
+}
+```
+
+~> **NOTE:** At this point, your Kubernetes cluster will have running masters
+and worker nodes, _however_, the worker nodes will not be able to join the
+Kubernetes cluster quite yet. The next section has the required Kubernetes
+configuration to enable the worker nodes to join the cluster.
+
+#### Required Kubernetes Configuration to Join Worker Nodes
+
+The EKS service does not provide a cluster-level API parameter or resource to
+automatically configure the underlying Kubernetes cluster to allow worker nodes
+to join the cluster via AWS IAM role authentication.
+
+While managing the underlying Kubernetes cluster configuration is beyond the
+scope of this guide, we provide an example of how to apply the required
+Kubernetes `ConfigMap` via `kubectl` below for completeness.
```

Comment:
A link to Config Map would be nice to have here.
https://cloud.google.com/kubernetes-engine/docs/concepts/configmap

Reply:
http://kubernetes.io/docs/user-guide/configmap/ would be better IMO

Reply:
Awesome! Will add.

Reply:
FYI we'll update the link to the one from @errm post-merge 👍